Libraries/LibJS/Runtime/RegExpPrototype.cpp

```diff
@@ -43,6 +44,18 @@ void RegExpPrototype::initialize(GlobalObject& global_object)
     Object::initialize(global_object);
     u8 attr = Attribute::Writable | Attribute::Configurable;
     define_native_function(vm.names.toString, to_string, 0, attr);
+    define_native_function(vm.names.test, test, 1, attr);
+    define_native_function(vm.names.exec, exec, 1, attr);
+
+    u8 readable_attr = Attribute::Configurable;
+    define_native_property(vm.names.dotAll, dotAll, nullptr, readable_attr);
```

Comment:
Please rename, for the sake of consistency:

```suggestion
    define_native_property(vm.names.dotAll, dot_all, nullptr, readable_attr);
```